plugin/src/main/java/com/denizenscript/denizen/scripts/commands/item/InventoryCommand.java

```diff
@@ -472,14 +426,23 @@ public void execute(final ScriptEntry scriptEntry) {
                     }
                     break;
                 case ADJUST:
+                    if (dataAction == null) {
+                        Debug.echoError("Inventory adjust must have a mechanism!");

```

Comment:
`throw new InvalidArgumentsRuntimeException` so it doesn't just NPE instantly on the next line